* Highly accurate protein structure prediction with AlphaFold



Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort
the structures of around 100000 unique proteins have been deteremined.

Coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches
are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will
 adopt based solely on its amino acid structure - the structure prediction component of the protien folding problem has been an important open reseacch problem for
 more than 50 years.

 Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein
 structure, leveraging multi-sequence alignments, into the design of the deep learning algotith
 

 
---

Despite these advances, contemporary physical and evolutionary history based approaches produce predictions that are far short
of experimental accuracy in the majority of cases in which a homologue has not been solved experimentally and this has limited
their utility for many biological applications


---

*The neural network alphafold that we developed was entered into the CASP14 assessment. The CASP assessment is carried out biennially
using recently solved structures that have not been deposited in the PDB or publically disclosed so that it is a blind test for the participating
methods, and has long served as the gold-standard assessment for the accuracy of structure prediction*




